docs/core/tools/project-json-to-csproj.md

```diff
@@ -151,6 +168,24 @@ Use the `TargetFrameworks` property to define your list of target frameworks. Us
 </PropertyGroup>
 ```
 
+### Microsoft.NETCore.App metapackage
+
+```json
+{
+  "dependencies": {
+    "Microsoft.NETCore.App": "1.0.0"
+  }
+}
+```
+
+```xml
+<PropertyGroup>
+  <RuntimeFrameworkVersion>1.0.3</RuntimeFrameworkVersion>
+</PropertyGroup>
+```
+
+Note that the `<RuntimeFrameworkVersion>` changed to the latest patch version installed in the machine for version 1.0.x.
```

Comment:
This is not true. The version we migrate too is actually coded in the migrate code itself. This means that, if you have 1.0.5 runtime install in the machine, we will not move you to that in your csproj, unless of course, such change happened in the migrate code itself.